v1.0.0

RafaelGSS opened the pull request:
- Add github deployment workflow
- Add heapprofiler documentation

---

TODO: Before it lands, we should configure the staging environment properly.

RafaelGSS committed:
chore: add deployment setup

RafaelGSS committed:
feat: add heap profiler

RafaelGSS commented:
> TODO: Before it lands, we should configure the staging environment properly.

Well, it looks like it's already been set up correctly.

simoneb commented:
A few Q:

- Are we planning to remove the private repo after this? 
- Did you try if the deployment to prod worked if run locally? Did you make changes in AWS so the deployment works from this repo?

RafaelGSS committed:
chore: trigger deployment to test

RafaelGSS commented:
> Are we planning to remove the private repo after this?

Yes

> Did you try if the deployment to prod worked if run locally? Did you make changes in AWS so the deployment works from this repo?

Yes, I have changed the OIDC rules, but I have changed the workflow just to make sure.

simoneb commented on .github/workflows/deploy.yml:
```diff
@@ -0,0 +1,41 @@
+name: Deploy
+
+# on:
+#   push:
+#     branches:
+#       - master
+on:
+  pull_request:
```
remember to restore

RafaelGSS committed:
chore: restore deployment on main

RafaelGSS commented:
@mcollina There is also a pre-configured Netlify deployment in another account, we don't know exactly who has created it, do you know? https://app.netlify.com/sites/vigorous-turing-f48ff5/deploys
